lib/members_page.rb

```diff
@@ -1,7 +1,12 @@
 # frozen_string_literal: true
-require_rel 'page'
+require 'scraped_page'
+
+class MembersPage < ScrapedPage
+
```

Comment:
A ScrapedPage subclass with no fields is a red flag. Aren't these methods part of the public API to this page?